cbt_enable_and_snapshot.py

```diff
@@ -20,6 +24,13 @@
 import argparse
 
 
+def enable_nbd_on_all_networks(session):
+    connection_type = "nbd"
+    network_list = (session.xenapi.network.get_all())[0].split(",")

```

Comment:
This is wrong. Sorry for not spotting this earlier.

session.xenapi.network.get_all() will return references to all networks.
[0] will return the first reference
split() will convert this back into an array.

II think you just want session.xenapi.network.get_all() to get all networks?!

Reply:
Yes you are right. Copied this code from my XenRT test and seeing as it didn't fail I had just assumed it was working. Fixed now.